medicai/utils/general.py

```
@@ -1,9 +1,161 @@
+
+        # Machine-friendly dict version
+        desc = {"name": name, "doc": base_doc}
+        if hasattr(cls, "ALLOWED_BACKBONE_FAMILIES"):
+            desc["allowed_backbones"] = cls.ALLOWED_BACKBONE_FAMILIES
+        try:
+            desc["args"] = {
+                pname: str(param)
+                for pname, param in inspect.signature(cls.__init__).parameters.items()
+                if pname != "self"
+            }
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)